tests/distributions/test_transform.py

```diff
@@ -345,56 +345,13 @@ def check_transform_elementwise_logp(self, model):
             .sum()
```

Comment:
Are you sure that you want to `sum` here? You might want to actually compare shapes.

Reply:
Shapes must match for the `transform_logp_eval` to work in the first place. Notice we checked the ndim is correct already.